esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -307,74 +373,44 @@ bool CWsESDLConfigEx::onPublishESDLDefinition(IEspContext &context, IEspPublishE
                     }
                     else
                     {
+                        // Outstanding question here. The current behavior is to return everything
+                        // starting with the <Definition> element as root. However, since it's returned
+                        // in the SOAP element named <Interface>, I think the intention may have been
+                        // to return with the root <esxdl>
+                        this->wrapWithDefinitionElement(defInfo, definitionxml);
```

Comment:
Let's discuss...

Reply:
As per our discussion, we'll leave this as-is. It could be considered important for the GetESDLDefinition SOAP method to return a *full* DESDL Definition that could, for example, be used as input to publish that definition on another deployment. If required in the future we may consider options to the method and an updated interface that would provide the definition in a different format (eg stripped of the <Definition> element). Additionally, any change to this interface would need to be versioned because of ECLWatch's use.